### DISCUSSION TITLE
change ALT +C bindkey

### DISCUSSION THREAD

**Discussion by zoggy23** [junegunn/fzf#3144] [Q&A] [answered]
Hi everyone, 

Can anyone guide me to change ALT +C bindkey for something else like ALT + P for example ? 

Thanks

**Comment by Ioulan** [comment]
What shell are you using? 

You can change it's key binds by overriding it.

On **zsh**
```
zle     -N     fzf-cd-widget
bindkey '^[p'  fzf-cd-widget
```
On **bash**
```
bind -x '"\ep": "__fzf_cd__"'
```

Simply append this on your `*.rc`
Let me know it if works.

**Comment by zoggy23** [comment]
Thanks for your response 
I use oh my zsh 

Thanks

**Comment by zoggy23** [comment, marked as answer]
I got solved by adding this line 
bindkey '^Y' fzf-cd-widget

Thanks for the help
